src/main/java/Neon.java

```diff
@@ -0,0 +1,244 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.Scanner;
+import java.io.FileWriter;
+import java.time.LocalDate;
+
+
+public class Neon {
+    private static final String NAME = "Neon";
+    private static final String DASH_BREAK = "-----------------------------------";
+    private static ArrayList<Task> list = new ArrayList<>();
+    private static int lastListIndex = 0;
+
+    private static void greetingLine() {
+        System.out.println(DASH_BREAK);
```

Comment:
Maybe you can write a `printDashBreak()` to abstract this out. This can make your code cleaner!